core/tests/webdriverio_utils/forms.js

```diff
@@ -119,7 +119,7 @@ var ListEditor = function(elem) {
   // NOTE: this returns a promise, not an integer.
   var _getLength = async function() {
     var items =
-      await $$(
+      await elem.$$(
```

Comment:
Why ele? Ditto elsewhere.

Reply:
This is the correct code, previously it was wrong. PTAL at the protractor code [here](https://github.com/oppia/oppia/blob/develop/core/tests/protractor_utils/forms.js#L124).